phssh_connector.py

```diff
@@ -957,22 +933,22 @@ def _block_ip(self, param):
         if root:
             passwd = None
         if not root and passwd is None:
-            return action_result.set_status(phantom.APP_ERROR, SSH_ERR_NEED_PW_FOR_ROOT)
+            return action_result.set_status(phantom.APP_ERROR, SSH_NEED_PW_FOR_ROOT_ERR)
 
         protocol = param[SSH_JSON_PROTOCOL]
         direction = "INPUT" if param[SSH_JSON_DIRECTION].lower() == "in" else "OUTPUT"
 
         try:
             if direction == "INPUT":
-                remote_ip = "-s {}".format(param[SSH_JSON_REMOTE_IP])
+                remote_ip = "-s {}".format(param.get(SSH_JSON_REMOTE_IP))
```

Comment:
Check for the None value. Remove try...except block.

Reply:
made required changes